domain/src/main/kotlin/no/nav/su/se/bakover/domain/Copyable.kt

```diff
@@ -11,4 +11,20 @@ sealed class CopyArgs {
         data class NyPeriode(val periode: Periode) : Tidslinje()
         object Full : Tidslinje()
     }
+
+    data class MaksPeriode(val periode: Periode) : CopyArgs() {
```

Comment:
Jeg skjønte ikke helt hva denne gjorde ut ifra navnet. Såvidt jeg klarer å skjønne så "lager" den en ny periode som overlapper de to opprinnelige (`preiode` og `original`)? Hadde det vært feil om den het `Intersection` ellerno?

Reply:
Snitt/Intersection er vel korrekt mattebegrep, men skjønner vi hva det betyr da etterpå (og at det ikke da betyr gjennomsnitt eller veikryss)?